### PR TITLE
Release notes

### PR DESCRIPTION
**Pull request by snowystinger**
Closes 

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

### CODE REVIEW

File: packages/dev/docs/pages/releases/2023-12-18.mdx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+As a parting holiday gift, check out our new landing page for React Aria Components, replete with interactive examples! /* INSERT LINK HERE */ Happy coding and see you in the new year! ☃️
+
+
+### React Aria Components API updates
+
+In this release, we made a few last API updates to React Aria Components as they move along to GA.
+
+We had to temporarily remove nested column support in Tables due to some last minute use cases which we deemed to be inconsistent. We wouldn't have found these without all of your help using our components. Nested column support will be back, and should be easier to use in the future. Thanks for understanding.
+
+/* Decide, do we mention removed onCellAction */
```
Comment (snowystinger) [inline packages/dev/docs/pages/releases/2023-12-18.mdx:44]: question, was this ever in use? can we just remove?

Comment (devongovett) [reply packages/dev/docs/pages/releases/2023-12-18.mdx:44]: I think we remove

File: packages/dev/docs/pages/releases/2023-12-18.mdx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+---
+description: In this release, we’re promoting React Aria Components from RC to GA 🎉, optimizing our included translation files, shipping a storybook based on all our React Aria Components with a starter theme, and fixing plenty of bugs.
+
+date: 2023-12-18
+---
+
+# December 18, 2023 Release
+
+Welcome to the last release of the year! In this release, we’re promoting React Aria Components from RC to GA 🎉, promoting React Spectrum's Dropzone to RC, introducing a brand new StepList to React Spectrum, adding an option to optimize our included translation files, shipping a storybook based on all our React Aria Components with a starter theme, and fixing plenty of bugs.
```
Comment (devongovett) [inline packages/dev/docs/pages/releases/2023-12-18.mdx:21]: Are we including StepList? Usually we don't mention alphas here. Maybe in the under construction section

File: packages/dev/docs/pages/releases/2023-12-18.mdx [hunk 3]
```diff
@@ -59,14 +59,18 @@ We had to temporarily remove nested column support in Tables due to some last mi
 * Add support for onScroll event to RAC collection components - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/5534)
 * Add hover state to `TableColumnHeader` when 'allowSorting' is true - [@sookmax](https://github.com/sookmax) - [PR](https://github.com/adobe/react-spectrum/pull/5471)
 * Use owner document in FocusScope - [@slye-stripe](https://github.com/slye-stripe) - [PR](https://github.com/adobe/react-spectrum/pull/5449)
+* Change RAC style merge orders [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/5506)
```
Comment (snowystinger) [inline packages/dev/docs/pages/releases/2023-12-18.mdx:61]: I left this one out intentionally, it's covered by `RAC support overriding default styles in overlay components`

File: packages/dev/docs/pages/releases/2023-12-18.mdx [hunk 4]
```diff
@@ -89,28 +93,33 @@ We had to temporarily remove nested column support in Tables due to some last mi
...
-* tailwindcss-react-aria-components to use require instead of import - [@tom-fletcher](https://github.com/tom-fletcher) - [PR](https://github.com/adobe/react-spectrum/pull/5541)
-* hover in RAC ListBox and remove onCellAction - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/5580)
-* SearchField escape key propagates #5480 - [@ktabors](https://github.com/ktabors) - [PR](https://github.com/adobe/react-spectrum/pull/5571)
-* restoreTextSelection on pointerup event - [@sookmax](https://github.com/sookmax) - [PR](https://github.com/adobe/react-spectrum/pull/5561)
+* Tailwind CSS plugin in RAC to use require instead of import - [@tom-fletcher](https://github.com/tom-fletcher) - [PR](https://github.com/adobe/react-spectrum/pull/5541)
+* Hover in RAC ListBox and remove onCellAction - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/5580)
+* Propgate SearchField escape key #5480 - [@ktabors](https://github.com/ktabors) - [PR](https://github.com/adobe/react-spectrum/pull/5571)
```
Comment (snowystinger) [inline packages/dev/docs/pages/releases/2023-12-18.mdx:96]:
```suggestion
* Propagate SearchField escape key #5480 - [@ktabors](https://github.com/ktabors) - [PR](https://github.com/adobe/react-spectrum/pull/5571)
```

File: packages/dev/docs/pages/releases/2023-12-18.mdx (hunk 1 above)
Comment (snowystinger) [reply packages/dev/docs/pages/releases/2023-12-18.mdx:44]:
```suggestion

```

File: packages/dev/docs/pages/releases/2023-12-18.mdx [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+
+---
+description: In this release, we’re promoting React Aria Components from RC to GA 🎉, optimizing our included translation files, shipping a storybook based on all our React Aria Components with a starter theme, and fixing plenty of bugs.
+
+date: 2023-12-18
+---
+
+# December 18, 2023 Release
+
+Welcome to the last release of the year! In this release, we’re promoting React Aria Components from RC to GA 🎉, promoting React Spectrum's Dropzone to RC, adding an option to optimize our included translation files, shipping a storybook based on all our React Aria Components with a starter theme, and fixing plenty of bugs.
```
Comment (devongovett) [inline packages/dev/docs/pages/releases/2023-12-18.mdx:21]:
```suggestion
Welcome to the last release of the year! In this release, we’re promoting React Aria Components from RC to GA 🎉, promoting React Spectrum's DropZone to RC, adding an option to optimize our included translation files, shipping a storybook based on all our React Aria Components with a starter theme, and fixing plenty of bugs.
```

File: packages/dev/docs/pages/releases/2023-12-18.mdx [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+
+As always, huge thanks to everyone in our community for their feedback and contributions to make this release and this year possible!
+
+In a big year, there are a few accomplishments we wanted to highlight.
+ - ESM support
+ - React StrictMode support
+ - React Aria Components
+ - React Spectrum new components, Dropzone, InlineAlert, TagGroup, ActionBar, Avatar, Table Col Resizing
```
Comment (devongovett) [inline packages/dev/docs/pages/releases/2023-12-18.mdx:29]:
```suggestion
 - React Spectrum new components, DropZone, InlineAlert, TagGroup, ActionBar, Avatar, Table Column Resizing
```

File: packages/dev/docs/pages/releases/2023-12-18.mdx [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+ - Docs Search
+ - Built-in support for form validation
+ - Native links support for collections
+ - Tailwind plugin for RAC
```
Comment (devongovett) [inline packages/dev/docs/pages/releases/2023-12-18.mdx:33]: We might want to order these by "exciting-ness"? While very nice, I wouldn't say ESM support was our most important feature this year 😉

File: packages/dev/docs/pages/releases/2023-12-18.mdx [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+As a parting holiday gift, check out our new landing page for [React Aria Components](../react-aria/index.html), replete with interactive examples! Happy coding and see you in the new year! ☃️
+
+
+### React Aria Components API updates
+
+In this release, we made a few last API updates to React Aria Components as they move along to GA.
+
+We had to temporarily remove nested column support in Tables due to some last minute use cases which we deemed to be inconsistent. We wouldn't have found these without all of your help using our components. Nested column support will be back, and should be easier to use in the future. Thanks for understanding.
+
+
```
Comment (devongovett) [inline packages/dev/docs/pages/releases/2023-12-18.mdx:44]: There were a few others as well.

File: packages/dev/docs/pages/releases/2023-12-18.mdx [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+ - Tailwind plugin for RAC
+
+As a parting holiday gift, check out our new landing page for [React Aria Components](../react-aria/index.html), replete with interactive examples! Happy coding and see you in the new year! ☃️
+
+
+### React Aria Components API updates
+
+In this release, we made a few last API updates to React Aria Components as they move along to GA.
+
+We had to temporarily remove nested column support in Tables due to some last minute use cases which we deemed to be inconsistent. We wouldn't have found these without all of your help using our components. Nested column support will be back, and should be easier to use in the future. Thanks for understanding.
```
Comment (devongovett) [inline packages/dev/docs/pages/releases/2023-12-18.mdx:42]: I would shorten this to just like "Temporarily removed nested column support in Tables due to instability." Not sure we need to explain ourselves all that much in the release notes.

File: packages/dev/docs/pages/releases/2023-12-18.mdx (hunk 7 above)
Comment (snowystinger) [reply packages/dev/docs/pages/releases/2023-12-18.mdx:33]: haha true
I was also thinking of trying to spice them up with something else, I thought image, but that'd be hard... any ideas?

File: packages/dev/docs/pages/releases/2023-12-18.mdx (hunk 8 above)
Comment (snowystinger) [reply packages/dev/docs/pages/releases/2023-12-18.mdx:44]: I thought that, but I didn't see anything else removed in the TSDiff, so what else do we need to call out?

Comment (snowystinger) [reply packages/dev/docs/pages/releases/2023-12-18.mdx:51]: I've added everything else I could think of

File: packages/dev/docs/pages/releases/2023-12-18.mdx (hunk 7 above)
Comment (LFDanLu) [reply packages/dev/docs/pages/releases/2023-12-18.mdx:32]: I think the intl string optimization is worth calling out, currently sweeping though the past releases

File: packages/dev/docs/pages/releases/2023-12-18.mdx (hunk 8 above)
Comment (LFDanLu) [reply packages/dev/docs/pages/releases/2023-12-18.mdx:51]: We should probably call out the dep array for RAC Table: https://github.com/adobe/react-spectrum/pull/5530

File: packages/dev/docs/pages/releases/2023-12-18.mdx [hunk 10]
```diff
@@ -0,0 +1,135 @@
...
+* Fix broken links in the the docs - [@ktabors](https://github.com/ktabors) - [PR](https://github.com/adobe/react-spectrum/pull/5572)
+* Update collections and selection guides for React Aria Components - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/5455)
+
+## Under construction
+
+Fixes to pre-released components are listed below. Please feel free to try them out, and report any issues you encounter.
+
+**React Spectrum Submenus (Alpha)**
+
+* React Spectrum Submenu support - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/4976)
```
Comment (LFDanLu) [inline packages/dev/docs/pages/releases/2023-12-18.mdx:127]:
```suggestion
* React Spectrum Submenu support - [@LFDanLu](https://github.com/LFDanLu) [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/4976)
```

File: packages/dev/docs/pages/releases/2023-12-18.mdx (hunk 8 above)
Comment (LFDanLu) [reply packages/dev/docs/pages/releases/2023-12-18.mdx:51]: oh do we also want to call out the `UNSTABLE_portalContainer`? Maybe not cuz unstable?

File: packages/dev/docs/pages/releases/2023-12-18.mdx [hunk 11]
```diff
@@ -0,0 +1,135 @@
...
+
+In this release, we made a few last API updates to React Aria Components as they move along to GA.
+
+ - Temporarily removed nested column support in Tables due to instability.
+ - Our Collections have come a long ways since we started RAC, please see the new page, [React Aria Collections](../react-aria/collections) for more information.
+ - Hover and Focus are now distinct states in ListBox.
+ - Hover state has been added to Column Headers in Tables when sorting is allowed.
+ - All refs given to a component now go to their outer most element. If you need a ref to the input, you can attach it directly.
+ - Styles for Popover, Tooltip, and OverlayArrow are now merged in a different order to allow overriding our default styles.
+ -
```
Comment (ktabors) [inline packages/dev/docs/pages/releases/2023-12-18.mdx:48]: Are we still adding these and this will get removed?

File: packages/dev/docs/pages/releases/2023-12-18.mdx [hunk 12]
```diff
@@ -0,0 +1,136 @@
...
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {BlogPostLayout, Hero} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+description: In this release, we’re promoting React Aria Components from RC to GA 🎉, optimizing our included translation files, shipping a storybook based on all our React Aria Components with a starter theme, and fixing plenty of bugs.
+
+date: 2023-12-18
```
Comment (yihuiliao) [inline packages/dev/docs/pages/releases/2023-12-18.mdx:16]: reminder to change the date for release

File: packages/dev/docs/pages/releases/2023-12-18.mdx [hunk 13]
```diff
@@ -0,0 +1,136 @@
...
+* Propagate SearchField escape key - [@ktabors](https://github.com/ktabors) - [PR](https://github.com/adobe/react-spectrum/pull/5571)
+* restoreTextSelection on pointerUp event - [@sookmax](https://github.com/sookmax) - [PR](https://github.com/adobe/react-spectrum/pull/5561)
+* Make value and defaultValue valid in number and slider fields - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/5543)
+* Respect Tailwind hoverOnlyWhenSupported, and fix placeholder-shown with Input - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/5587)
+* Top margin of a button in a form - [@yihuiliao](https://github.com/yihuiliao) - [PR](https://github.com/adobe/react-spectrum/pull/5461)
+* TableView resize observer loop limit - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/5432)
+* TableView virtualizer crash - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/5380)
+
```
Comment (yihuiliao) [inline packages/dev/docs/pages/releases/2023-12-18.mdx:109]: add https://github.com/adobe/react-spectrum/pull/5603

File: packages/dev/docs/pages/releases/2023-12-20.mdx [hunk 14]
```diff
@@ -0,0 +1,277 @@
...
+
+Happy coding and see you in the new year! ☃️
+
+
+### React Aria Components API updates
+
+In this release, we made a few last API updates to React Aria Components as they move along to GA.
+
+ - Temporarily removed nested column support in `Tables` due to instability.
+ - Our `Collections` have come a long ways since we started RAC, please see the new page, [React Aria Collections](../react-aria/collections) for more information.
```
Comment (devongovett) [inline packages/dev/docs/pages/releases/2023-12-20.mdx:34]: This link doesn't work.

```suggestion
 - Our `Collections` have come a long ways since we started RAC, please see the new page, [React Aria Collections](../react-aria/collections.html) for more information.
```

File: packages/dev/docs/pages/releases/2023-12-20.mdx [hunk 15]
```diff
@@ -0,0 +1,277 @@
...
+ - Hover and Focus are now distinct states in `ListBox`.
+ - Hover state has been added to `Column Headers` in `Tables` when sorting is allowed.
```
Comment (devongovett) [inline packages/dev/docs/pages/releases/2023-12-20.mdx:36]:
```suggestion
 - Hover state has been added to column headers in tables when sorting is allowed.
```

File: packages/dev/docs/pages/releases/2023-12-20.mdx [hunk 16]
```diff
@@ -0,0 +1,277 @@
...
+
+---
+description: In this release, we’re promoting React Aria Components from RC to GA 🎉, optimizing our included translation files, shipping a storybook based on all our React Aria Components with a starter theme, and fixing plenty of bugs.
+
+date: 2023-12-20
+---
+
+# December 20, 2023 Release
+
+Welcome to the last release of the year! After a year of work, we are happy to announce the GA release of React Aria Components 🎉 This component library, built on top of our React Aria hooks, provides a simpler way to craft world-class accessible components with custom styles. See our new React Aria [landing page](https://react-spectrum.adobe.com/react-aria/index.html) for all of the details, including interactive examples. To help get you started even faster, we now also have a downloadable Storybook [starter kit](../react-aria/getting-started#starter-kit) including an example CSS theme.
```
Comment (devongovett) [inline packages/dev/docs/pages/releases/2023-12-20.mdx:21]: you need the .html

```suggestion
Welcome to the last release of the year! After a year of work, we are happy to announce the GA release of React Aria Components 🎉 This component library, built on top of our React Aria hooks, provides a simpler way to craft world-class accessible components with custom styles. See our new React Aria [landing page](https://react-spectrum.adobe.com/react-aria/index.html) for all of the details, including interactive examples. To help get you started even faster, we now also have a downloadable Storybook [starter kit](../react-aria/getting-started.html#starter-kit) including an example CSS theme.
```

File: packages/dev/docs/pages/releases/2023-12-20.mdx (hunk 14 above)
Comment (snowystinger) [reply packages/dev/docs/pages/releases/2023-12-20.mdx:34]: no, but figured if anyone was coming from alpha.1 or something, might be good to call out

**Review by devongovett — DISMISSED** [review]

**Review by dannify — APPROVED** [review]
